.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: java
+
+jdk:
+  - oraclejdk7
+  - openjdk7
+
+env:
+  global:
+    - secure: "dcVTtJxm7Rwt1RvrZaKVs4y8yUNPQlzmP+NwyV6dZ2IpO/5iaioK9MlUa0YfA+jZwdRN94X7BWKfA30GJ8fUfCYSqcn18jInKJwwFZhwfVS0F4hYw2ascyO3beQm+OHaEatdyZoDQGlxyeVIL4eAkTy74tzooMVG4m9LN6F9ipA="
```

Comment:
Yep. Its not actually the key but standard functionality in travis to encrypt a sensitive key. <https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables>. Thanks for double checking though 😁 !